draft-ietf-scone-protocol.md

```diff
@@ -281,10 +286,12 @@ limiting is necessary.  Alternatively, a receiver can control the release of
 flow control credit (see {{Section 4 of QUIC}}) to indirectly limit the sending
 rate of a peer.
 
-Some applications offer options for rate control that can offer superior
-outcomes.  Real-time and streaming video applications are able to adjust video
-quality to fit within a target throughput.  For instance, an HTTP Live Streaming
-client {{?HLS=RFC8216}} can ask for lower bitrate, lower quality media segments.
+Some applications offer options for rate control that can offer superior outcomes.
+Applications that are able to adapt bit rates,
+such as real-time and streaming video applications,
+can adjust video quality to fit within a target throughput.
```

Comment:
Seems slightly clunky. It could be something alonmg the lines of: "Video-based applications that can adjust video quality, such as real-time and streaming video applications,
can adjust their rate to fit within a target throughput"